docs/concepts/jcloud/index.md

```diff
@@ -482,6 +482,219 @@ jc job logs myjob1 -f rich-husky-af14064067
 :width: 90%
 ```
 
+## Deployments
+
+### Deploy
+
+In Jina's idiom, a project is a [Deployment](https://docs.jina.ai/concepts/orchestration/Deployment/), which represents an end-to-end task such as indexing, searching or recommending. In this document, we use "project" and "Deployment" interchangeably.
```

Comment:
Avoid re-defining the term, just act as it is already defined.

Reply:
```suggestion
A Jina [Deployment](https://docs.jina.ai/concepts/orchestration/Deployment/) represents an end-to-end task such as indexing, searching, or recommending. In this document, we use "project" and "Deployment" interchangeably.
```